### PR TITLE
WIP: Upgrade-guide for Plone 5.1

### PR DESCRIPTION
**Pull request by pbauer**
Fixes: https://github.com/plone/Products.CMFPlone/issues/2127

### CODE REVIEW

File: manage/upgrading/version_specific_migration/upgrade_to_51.rst [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+* `Portal actions control panel <https://github.com/plone/Products.CMFPlone/issues/1342>`_
+* `Add direct link from a group name on Sharing tab to that group's member list <https://github.com/plone/Products.CMFPlone/issues/1310>`_
+* `Remove plone.app.openid from core <https://github.com/plone/Products.CMFPlone/issues/1659>`_
+* `Get rid of portal_quickinstaller <https://github.com/plone/Products.CMFPlone/issues/1340>`_
+* `Add support for conditionally import registry records  <https://github.com/plone/Products.CMFPlone/issues/1406>`_
+* `Auto-Rotation for Images <https://github.com/plone/Products.CMFPlone/issues/1673>`_
+* `assimilate collective.indexing <https://github.com/plone/Products.CMFPlone/issues/1343>`_
+* Use lxml cleaner for savehtml transforms
+* `Easily change default search order <https://github.com/plone/Products.CMFPlone/issues/1600>`_
+* `Retina image scales <https://github.com/plone/Products.CMFPlone/issues/1483>`_
```
Comment (hvelarde) [inline manage/upgrading/version_specific_migration/upgrade_to_51.rst]: let's get rid of the Retina term as it's a trademark of Apple.

**Comment by svx** [comment]
I adjusted it according to: https://github.com/plone/documentation/blob/5.1/about/contributing/word_choice.rst :)

**Comment by svx** [comment]
@pbauer @hvelarde Can you ping me when you think I can review and/or merge ?

Thanks !

**Review by hvelarde — APPROVED** [review]
LGTM

**Review by svx — APPROVED** [review]
LGTM

**Comment by svx** [comment]
Done !

Thank you !

**Comment by pbauer** [comment]
Thank you!

**Pull request merged**